style.css

```diff
@@ -0,0 +1,81 @@
+.credit-card {
+    background-color: #425ecd;
+    height: 204px;
+    display: flex;
+    flex-direction: column;
+    font-size: 20px;
+    border-radius: 10px;
+    margin-bottom: 20px;
+    font-family: "Arial";
+}
+
+.credit-card-front {
+    width: 324px;
+    display: flex;
+    justify-content: flex-end;
+    padding-left: 20px;
+}
+
+.number {
+
+}
```

Comment:
When you create a PR the first thing you should do is self-review. If you would do it this kind of code would not be here ;)

Reply:
You are right, I'm sorry about that!